doc/design/fluid_dist/parameter_server.md

```diff
@@ -59,6 +59,13 @@ After converting:
      queue. It will block until the queue has the required number of
      tensors.
 
+### Sparse Update
+
+For an embedding layer, the gradient maybe be very sparse(upper 90% is zero) for each mini-batch.
```

Comment:
There should be a space between sparse and `(`.

Reply:
upper => up to

Reply:
the gradient maybe be very sparse => the gradient may have many rows containing only 0

Reply:
Done.